package.json

```diff
@@ -27,5 +27,13 @@
         "typescript": "^5.5.4",
         "vitest": "^2.1.1"
     },
+    "resolutions": {
+        "@swup/prettier-config": "link:node_modules/.cache/null",
+        "@swup/browserslist-config": "link:node_modules/.cache/null",
+        "microbundle": "link:node_modules/.cache/null",
+        "prettier": "link:node_modules/.cache/null",
+        "shelljs": "link:node_modules/.cache/null",
+        "shelljs-live": "link:node_modules/.cache/null"
```

Comment:
I wasn't able to configure Yarn resolutions to be more precise about what dependencies I want to remove, something like `"@swup/plugin/microbundle": "link:node_modules/.cache/null"`.
Do to that, I think we must upgrade to @yarnpkg/cli

Reply:
https://github.com/symfony/ux/pull/2222 :)